extensions/airport/src/index.tsx

```diff
@@ -0,0 +1,149 @@
+        new: appsResponse?.apps.all.filter(
+          (app) =>
+            !appsResponse?.apps.popular.includes(app.id) &&
+            !appsResponse?.apps.featured.includes(app.id) &&
+            appsResponse?.apps.new.includes(app.id) &&
+            !appsResponse?.apps.upcoming.includes(app.id) &&
+            !appsResponse?.apps.waitlist.includes(app.id)
+        ),
+        upcoming: appsResponse?.apps.all.filter(
+          (app) =>
+            !appsResponse?.apps.popular.includes(app.id) &&
+            !appsResponse?.apps.featured.includes(app.id) &&
+            !appsResponse?.apps.new.includes(app.id) &&
+            appsResponse?.apps.upcoming.includes(app.id) &&
+            !appsResponse?.apps.waitlist.includes(app.id)
+        ),
+        waitlist: appsResponse?.apps.all.filter(
+          (app) =>
+            !appsResponse?.apps.popular.includes(app.id) &&
+            !appsResponse?.apps.featured.includes(app.id) &&
+            !appsResponse?.apps.new.includes(app.id) &&
+            !appsResponse?.apps.upcoming.includes(app.id) &&
+            appsResponse?.apps.waitlist.includes(app.id)
+        ),
+      }));
+    }
+    fetch();
+  }, []);
+
+  return (
+    <List isLoading={state.all.length === 0} searchBarPlaceholder="Filter apps by name...">
+      <List.Section id="popular" title="Popular">
+        {state.featured.map((app: App) => (
+          <AppListItem key={app.id} type="featured" app={app} />
+        ))}
+      </List.Section>
+      <List.Section id="featured" title="Featured">
+        {state.popular.map((app: App) => (
+          <AppListItem key={app.id} type="popular" app={app} />
+        ))}
+      </List.Section>
+      <List.Section id="new" title="New">
+        {state.new.map((app: App) => (
+          <AppListItem key={app.id} type="new" app={app} />
+        ))}
+      </List.Section>
+      <List.Section id="upcoming" title="Upcoming">
+        {state.upcoming.map((app: App) => (
+          <AppListItem key={app.id} type="upcoming" app={app} />
+        ))}
+      </List.Section>
+      <List.Section id="waitlist" title="Waitlist">
+        {state.waitlist.map((app: App) => (
+          <AppListItem key={app.id} type="waitlist" app={app} />
+        ))}
+      </List.Section>
+      <List.Section id="all" title="All">
+        {state.all.map((app: App) => (
+          <AppListItem key={app.id} type="all" app={app} />
+        ))}
+      </List.Section>
+    </List>
+  );
+}
+
+function AppListItem(props: { app: App; type: string }) {
+  const { app, type } = props;
+
+  return (
+    <List.Item
+      id={`${type}-${app.id}`}
+      key={`${type}-${app.id}`}
+      title={app.name}
+      subtitle={app.description}
+      icon={{ source: app.icon }}
+      actions={
+        <ActionPanel>
+          <OpenInBrowserAction url={`https://app.airport.community/app/${app.id}?utm_source=raycast`} />
```

Comment:
Do you really need this UTM parameter? We don't do this to respect the user's privacy. I know that this isn't user related but some folks don't like UTM either. Wanna remove it?